images/chromium-headful/wrapper.sh

```diff
@@ -179,12 +202,13 @@ if [[ "${WITH_KERNEL_IMAGES_API:-}" == "true" ]]; then
       done
 
       # Wait for kernel-images API port 10001 to be ready.
-      echo "Waiting for kernel-images API port 10001..."
-      while ! nc -z localhost 10001 2>/dev/null; do
+      echo "Waiting for kernel-images API port 127.0.0.1:10001..."
+      while ! nc -z 127.0.0.1 10001 2>/dev/null; do
```

Comment:
potential optimization: I'd expect the server to come up faster than chromium so we could probably trim some time by inverting the order